### DISCUSSION TITLE
无法正常使用，麻烦退款，谢谢

### DISCUSSION THREAD

**Discussion by 1174290471** [getlantern/lantern-forum-cn#940] [Q&A]
邮箱收不到验证码，无法升级为专业版，请退款
1174290471@qq.com
![账单支付截图](https://github.com/user-attachments/assets/7b0adb31-3f2c-4b6e-9257-40add72e33b9)

**Comment by MaxMKMax** [comment]
@1174290471 您好. 非常抱歉啊! 我帮您和技术人员拿验证码. 请您别退款. 我这里尽全力帮您弄好. 谢谢!

**Comment by MaxMKMax** [comment]
@1174290471 
您好. 这是您的激活码:
82XC4-W9G7Q-67QTK-RVRDP-Q3WJG

**Reply by 1174290471** [reply]
非常感谢，可以正常用了

**Comment by MaxMKMax** [comment]
@1174290471 不客气. 谢谢您支持蓝灯! 能帮到您我们很开心.
